### PR TITLE
added boxer step feat

### PR DESCRIPTION
**Pull request by bleehu**
modified:   CharacterCreation/03_Feats.txt

### CODE REVIEW

**Comment by bleehu** [comment]
Issue #926

File: CharacterCreation/03_Feats.txt [hunk 1]
```diff
@@ -83,6 +83,24 @@ sticky situation then you won't get to hear from him anyway.
 range bracket. For more information on hip firing, see the "Hip Firing" section
 of the Shooting Rules document in the Combat Rules directory.
 
+== Boxer Step ==
+
+Prerequisites:
+
+*  Dex > 3
+*  Unarmed Training
```
Comment (Turtlelord26) [inline CharacterCreation/03_Feats.txt]: Unarmed Combat Training

File: CharacterCreation/03_Feats.txt [hunk 2]
```diff
@@ -83,6 +83,24 @@ sticky situation then you won't get to hear from him anyway.
...
+
+    You've mastered the simple, but imperative art of quickly advancing in close
+combat. This makes it really hard for an enemy to create space between you to
+avoid your reach.
+
+    If an enemy tries to disengage, they must first make a Reflex save. If they
```
Comment (Turtlelord26) [inline CharacterCreation/03_Feats.txt]: We should specify "tries to disengage ^from melee combat with you^"

**Review by Turtlelord26 — CHANGES_REQUESTED** [review]
Very nice, I like this sentineloid feat.
See line items below

**Review by Turtlelord26 — APPROVED** [review]

**Pull request merged**